src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -1003,4 +1003,13 @@ public void changeToDesktopView() {
         browser.driver.manage().window().maximize();
     }
 
+    /**
+     * @return true if the element is in the user's visible area of a web page.
```

Comment:
Yes. I prefer to have the overview sentence rather than the `@return` tag though.
`Returns true if ...`
rather than
`@return true...`